pkg/util/kubernetes/kube_util.go

```diff
@@ -18,17 +18,37 @@ package kubernetes
 
 import (
 	"fmt"
+	"k8s.io/client-go/kubernetes"
+	"os"
 	"runtime"
 
 	"github.com/pkg/errors"
-
 	"k8s.io/client-go/discovery"
 	"k8s.io/client-go/rest"
 	"k8s.io/client-go/tools/clientcmd"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
```

Comment:
`logf`? why not `log`?

Reply:
I prefer not using "util" files. 
I suggest moving the code to a different file (`kube_client.go`?)

Reply:
this file already in use by ibmblockcsi_controller.go and it using a similar library so it seems right to be there.. (if you still think I should move it it's supposed to be a problem)
logf - its the convention in the code (more than 12 places name it like that)

Reply:
1. "util" files can be easily abused since we can define lots of things at "utilities". Thus, I prefer to reduce our "util" usage.
2. what is the meaning of `f` in `logf`?

Reply:
1. will move.
2. nor sure but it's in the [doc too](https://sdk.operatorframework.io/docs/building-operators/golang/references/logging/#a-simple-example)

Reply:
done